Makefile

```
@@ -211,7 +211,8 @@ test-unit: $(SETUP_ENVTEST) envtest-k8s-bins #HELP Run the unit tests
                 -tags '$(GO_BUILD_TAGS)' \
                 -cover -coverprofile ${ROOT_DIR}/coverage/unit.out \
                 -count=1 -race -short \
-                $(UNIT_TEST_DIRS) \
+                -run "$(if $(TEST_FILTER),$(TEST_FILTER),.)" \
+                $(if $(TEST_PKG),$(TEST_PKG),$(UNIT_TEST_DIRS)) \
```

Rather than both `UNIT_TEST_DIRS` and `TEST_PKG`, I think we should consolidate to avoid confusion about what to set. How about this?
```make
ifeq ($(origin TEST_PKGS), undefined)
TEST_PKGS := $(shell go list ./... | grep -v /test/ | grep -v /catalogd/test/)
endif
```

Then, this line can stay simple:
```suggestion
                $(TEST_PKGS) \
```

---

yeah that makes sense. done.